flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/functions/inference/LookupCallContext.java

```diff
@@ -35,14 +34,15 @@
 import java.util.Optional;
 
 import static org.apache.flink.table.functions.UserDefinedFunctionHelper.generateInlineFunctionName;
+import static org.apache.flink.table.planner.plan.utils.FunctionCallUtils.FieldRef;
 import static org.apache.flink.table.types.logical.utils.LogicalTypeChecks.getFieldTypes;
 import static org.apache.flink.table.types.utils.TypeConversions.fromLogicalToDataType;
 
 /** The {@link CallContext} of a {@link LookupTableSource} runtime function. */
 @Internal
 public class LookupCallContext extends AbstractSqlCallContext {
 
-    private final List<LookupKey> lookupKeys;
+    private final List<Variable> lookupKeys;
```

Comment:
I am curious. How we can replace the LookupKey type with Variable. But only the Constant and FieldRef extend Variable. LookupKey does not extend Variable.